model-api-gen/src/main/kotlin/org/modelix/metamodel/generator/MetaModelGenerator.kt

```diff
@@ -117,11 +122,41 @@ class MetaModelGenerator(val outputDir: Path, val nameConfig: NameConfig = NameC
             .write()
     }
 
+    private fun generateConceptMetaPropertiesInterface(languages: IProcessedLanguageSet) {
+        val fqName = checkNotNull(conceptPropertiesInterfaceName)
+        val interfaceName = ClassName(fqName.substringBeforeLast("."), fqName.substringAfterLast("."))
```

Comment:
Might need some error handling with good messages to the (Gradle) user. Right now, this probably fails in case the name does not contain a period?